content/200-orm/400-tools/05-prisma-cli.mdx

```diff
@@ -139,7 +139,7 @@ prisma
 <cmdResult>
 
 ```code no-copy
-Prisma is a modern DB toolkit to query, migrate and model your database (https://www.prisma.io)
+Prisma ORM is a modern DB toolkit to query, migrate and model your database (https://www.prisma.io)
```

Comment:
This change requires a similar change in `prisma/prisma` so the CLI actually outputs this :D

Reply:
Oops should I change back until/if that happens?